dotnet/src/SemanticKernel.Abstractions/Kernel.cs

```diff
@@ -318,6 +357,82 @@ public CultureInfo Culture
     }
     #endregion
 
+    #region Internal Filtering
+
+    [Experimental("SKEXP0005")]
```

Comment:
I think we should continue using the same experimental ID as for events. It's the same feature, just evolved with a new shape. Anyone currently using events will need to switch to the new syntax and shouldn't need to change the ID to do so.

Reply:
Makes sense, taking into account that in follow-up PR I will mark events as Obsolete, I think we can use same ID for filters. I updated the code.